.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,56 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What is the change in 5 words or less?'
+        default: ''
+        required: true
+      email:
+        description: 'Your terraformation email'
+        default: ''
+        required: true
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Create branch
+        run: git checkout -b `echo design-workflow-${{ github.actor }}-${{ github.event.inputs.description }} | sed "s/ /-/g"`
```

Comment:
maybe this branch naming is not a good idea.. using user input

Reply:
Maybe we can make user input a branch name?